native-maven-plugin/src/testFixtures/groovy/org/graalvm/buildtools/maven/IsolatedMavenExecutor.groovy

```diff
@@ -67,7 +67,7 @@ class IsolatedMavenExecutor {
         cliArgs.add("-cp")
         cliArgs.add(classpath)
         if (debug) {
-            cliArgs.add("-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=*:5005")
+            cliArgs.add("-agentlib:jdwp=transport=dt_socket,server=y,suspend=y,address=*:5005")
```

Comment:
Could you explain why this change is needed?

Reply:
This option is used for debugging. With this option enabled (`y`) process will wait for us to start the debugger. If the option is set to `n` you need to click debugger button (in IDE) multiple times to catch the moment when process allows you to attach the debugger (you don't know when it will be possible). I don't see a reason for this option to be `n`. If there is a need for that, please explain me.

Reply:
Maybe because this is used for test.

@melix Could you provide a feedback on this since I think you committed those lines of code?

Reply:
It is used in the tests, yes. When you want to attach a debugger, and suspend is `n`, you need to watch a gigantic log for the corresponding message. If you try too early, it will fail. If you try too late, you may have missed your mojo.

Suspending will basically wait for you to attach so that you don't miss the correct moment to do it.

Reply:
So should this value be changed from `n` to `y` as suggested above from your POV?

Reply:
Yes